### PR TITLE
Team/tired bunch/feature/pingpong boost

### PR DESCRIPTION
**Pull request by Ranko030**
I added an option wherre when you press space or 0 you speed up the ball and when you press again you slow it down . I also added an option when you press esc or backspace to exit the game

### CODE REVIEW

**Comment by DPrivalov** [comment]
don't forget to choose team/tired_bunch/main as a base branch. I changed it this time

File: source/dagger/gameplay/ping_pong/ping_pong_main.cpp [hunk 1]
```diff
@@ -37,7 +37,7 @@ void ping_pong::CreatePingPongBall(float tileSize_, ColorRGBA color_, Vector3 sp
     transform.position.z = pos_.z;
     auto& ball = reg.emplace<PingPongBall>(entity);
     ball.speed = speed_ * tileSize_;
-
+    /*if (rand() % 100 > 50)ball.speed *= -1;*///DODATO
```
Comment (DPrivalov) [inline source/dagger/gameplay/ping_pong/ping_pong_main.cpp]: something strange here

File: source/dagger/gameplay/ping_pong/pingpong_ball.h [hunk 2]
```diff
@@ -14,6 +14,7 @@ namespace ping_pong
         bool reachedGoal{ false };
         bool playerOneScored{ true };
         bool processed{ false };
+        bool boosted{ false };//ADDED
```
Comment (DPrivalov) [inline source/dagger/gameplay/ping_pong/pingpong_ball.h]: there is no need in comments like this: //ADDED
remove it

**Review by DPrivalov — CHANGES_REQUESTED** [review]
there is no need in that comments, remove them pls

File: source/dagger/gameplay/ping_pong/ping_pong_main.cpp (hunk 1 above)
Comment (Ranko030) [reply source/dagger/gameplay/ping_pong/ping_pong_main.cpp]: Sorry about that. Was trying to improvise for something else.

**Review by DPrivalov — APPROVED** [review]

**Review by dimiivko — APPROVED** [review]

**Pull request merged**